app/components/resend-invitation/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: '', // tagless component
+
+  store: Ember.inject.service(),
```

Comment:
The store is a service as of ember data 16, which we are using: http://emberjs.com/blog/2015/03/23/ember-data-1-0-beta-16-released.html#toc_store-as-a-service